packages/manager/apps/web/client/app/private-database/database/version/update/private-database-database-version-update.controller.js

```diff
@@ -82,5 +82,14 @@ angular.module('App').controller(
           );
         });
     }
+
+    getDatabaseDisplayName(value) {
+      const keyToTranslate = 'privateDatabase_dashboard_version_';
+      return this.privateDatabaseService.constructor.getDatabaseDisplayName(
+        this.$translate,
```

Comment:
Is `privateDatabaseService` vanilla JS ? `$translate` could be injected its in constructor instead of passing the instance